README.md

```diff
@@ -93,13 +93,21 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 ```javascript
 
-import { WechatyBuilder } from 'wechaty'
+import { WechatyBuilder } from "wechaty";
+
+const wechatyBot = WechatyBuilder.build({}); // get a Wechaty instance
+
+wechatyBot.on("scan", (qrcode, status) =>
+    console.log(
```

Comment:
They are still not in one line.

We need to put all code to one line just as same as before, thanks.